pkg/tsdb/sqleng/trim.go

```diff
@@ -0,0 +1,53 @@
+package sqleng
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+)
+
+// trim trims rows that are outside the qm.TimeRange
+func trim(f *data.Frame, qm DataQueryModel) error {
```

Comment:
We didn't have this before, why is it needed now? Something missing in the re-sample code that you have to take heights for in this trim function?